src/components/Input.jsx

```diff
@@ -1,5 +1,21 @@
-const Input = () => {
-  return <div>Input</div>;
+const Input = ({ value, onChange, name, placeholder, classes }) => {
+  return (
+    <div className={`flex items-center w-1/2 text-white ${classes}`}>
+      <p className="text-xs px-1 z-40 bg-code-black flex fixed ml-3 mb-[3%] select-none">
+        {name}
+      </p>
+      <div className="w-full">
```

Comment:
this should not be needed, the input component should be able to do it without the div